### PR TITLE
Added changes to talent show page and added Trivia party page

### PR DESCRIPTION
**Pull request by CatalinaAlbornoz**

### CODE REVIEW

File: jinja/carnival/attractions.html [hunk 1]
```diff
@@ -60,7 +60,8 @@ <h5 class="mb-1 black-text px-3">Nov 1-22</h5>
             </div>
            <div class="col-md-4 mb-5">
                     <div class="card card-image" style="background-image: url(../img/tent_blur.png); background-size: cover;">
-                        <div class="text-center d-flex align-items-center rgba-white-strong mask">
+                        <a href="trivia_party.html">
```
Comment (co9olguy) [inline jinja/carnival/attractions.html:63]: Don't forget to close the hyperlink! (`</a>`)

File: jinja/carnival/trivia_party.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">
+            A celebration needs a party so come to the Quantum Carnival's very own <strong>Trivia Party!</strong>
+            </h3>
+	<div style="font-size: 14pt;">
+            <p>
+            Are you ready to put your knowledge to the test?
+</p>
+
+<p>
+Come to the Trivia Party on November 12th at noon ET and have fun while learning new things!
```
Comment (ParhamPashaei) [inline jinja/carnival/trivia_party.html:23]: at noon (12pm ET)
or link to time converter:
https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250

File: jinja/carnival/attractions.html (hunk 1 above)
Comment (CatalinaAlbornoz) [inline jinja/carnival/attractions.html:63]:
```suggestion
                        <a href="trivia_party.html"></a>
```

File: jinja/carnival/trivia_party.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">
+            A celebration needs a party so come to the Quantum Carnival's very own <strong>Trivia Party!</strong>
+            </h3>
+	<div style="font-size: 14pt;">
+            <p>
+            Are you ready to put your knowledge to the test?
+</p>
+
+<p>
+Come to the Trivia Party on November 12th at noon ET and have fun while learning new things! Check the event at your local time zone <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250"> here </a>
```
Comment (CatalinaAlbornoz) [inline jinja/carnival/trivia_party.html:23]:
```suggestion
Come to the Trivia Party on November 12th at noon ET and have fun while learning new things! 
Check out the event at your local time zone <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250"> here </a>.
```

File: jinja/carnival/trivia_party.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            A celebration needs a party so come to the Quantum Carnival's very own <strong>Trivia Party!</strong>
+            </h3>
+	<div style="font-size: 14pt;">
+            <p>
+            Are you ready to put your knowledge to the test?
+</p>
+
+<p>
+Come to the Trivia Party on November 12th at noon ET and have fun while learning new things! 
+Check out the event at your local time zone <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250"> here </a>.
```
Comment (CatalinaAlbornoz) [inline jinja/carnival/trivia_party.html:24]:
```suggestion
<p>
See the event in your local time zone <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250"> here </a>.
</p>
```

File: jinja/carnival/trivia_party.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            </h3>
+	<div style="font-size: 14pt;">
+            <p>
+            Are you ready to put your knowledge to the test?
+</p>
+
+<p>
+Come to the Trivia Party on November 12th at noon ET and have fun while learning new things! 
+<p>
+See the event in your local time zone <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250"> here </a>.
```
Comment (co9olguy) [inline jinja/carnival/trivia_party.html:25]:
```suggestion
See the event in your local time zone <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Quantum+Carnival+Trivia&iso=20211112T12&p1=250"> here</a>.
```

**Review by co9olguy — APPROVED** [review]

**Pull request merged**